apps/backend/src/handlers/patchForm.ts

```diff
@@ -32,6 +32,11 @@ export const patchFormHandler = async (event: APIGatewayEvent) => {
   try {
     await createTableIfNotExists();
     await updateDocumentAdminNotes(id, JSONBody);
+      // Check if the "read" property is included in the request body
```

Comment:
Why do we need to do this here if we already have the other endpoing?

Also, this patchForm function seems to be pretty poorly named, but it seems like it's just for updating the admin notes, and updateDocumentAdminNotes expects the JSONBody to be an object that matches the adminNotesSchema, so read will never be in there.

Reply:
Looks like I forgot to remove it from there. I removed it and named this function better in my most recent commit.